dask/array/chunk.py

```diff
@@ -396,3 +396,13 @@ def slice_with_int_dask_array_aggregate(idx, chunk_outputs, x_chunks, axis):
             idx_final if i == axis else slice(None) for i in range(chunk_outputs.ndim)
         )
     ]
+
+
+def getitem(obj, index):
```

Comment:
Can you add a docstring here with a small comment about why we're adding the `copy`? This is a subtle point, so giving a bit of background will help future code readers